install_config/cluster_metrics.adoc

```diff
@@ -351,55 +351,64 @@ be added to your inventory file if it is necessary to override them.
 |Variable |Description
 
 |`openshift_metrics_install_metrics`
-|Deploy metrics if `true`, otherwise undeploy.
+|Deploy metrics if `true`. Otherwise, undeploy.
 
 |`openshift_metrics_start_cluster`
 |Start the metrics cluster after deploying the components.
 
 |`openshift_metrics_image_prefix`
-|The prefix for the component images. For
-example, with "openshift/origin-metrics-cassandra:v1.3", set prefix
-"openshift/origin-".
+|The prefix for the component images. For example, with
```

Comment:
Thanks! The ifdef statements make it render poorly the rich view, but it appears fine in a local build. This is what it will look like in the OCP docs:
http://file.rdu.redhat.com/~ahardin/03302017/update-metrics-ansible-variables/install_config/cluster_metrics.html